tests/HtmlConverterTest.php

```diff
@@ -133,6 +133,16 @@ public function test_anchor()
         $this->html_gives_markdown('<a href="ftp://files.example.com">ftp://files.example.com</a>', '<ftp://files.example.com>');
         $this->html_gives_markdown('<a href="mailto:test@example.com">test@example.com</a>', '<test@example.com>');
         $this->html_gives_markdown('<a href="mailto:test+foo@example.bar-baz.com">test+foo@example.bar-baz.com</a>', '<test+foo@example.bar-baz.com>');
+
+        // Autolinking can be toggled off
+        $markdown = new HtmlConverter();
+        $markdown->getConfig()->setOption('use_autolinks', false);
+        $result = $markdown('<a href="https://www.google.com">https://www.google.com</a>');
+        $this->assertEquals('[https://www.google.com](https://www.google.com)', $result);
+        $result = $markdown('<a href="https://www.google.com">Google</a>');
+        $this->assertEquals('[Google](https://www.google.com)', $result);
+        $result = $markdown('<a href="google.com">google.com</a>');
+        $this->assertEquals('[google.com](google.com)', $result);
```

Comment:
Would you mind changing this to use the `$this->html_give_markdown()` helper method instead for consistency?

Reply:
Ah! I totally missed the fact that this method also accepts options. There are so few test cases using it that at a first glance I missed that completely.